.github/workflows/tests.yaml

```
@@ -25,6 +25,18 @@ jobs:
         git checkout -b "$branch" || true
     - name: Install earthly
       run: "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/download/v0.6.23/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
+    - name: Configure pull-through-cache
+      run: |
+        mkdir ~/.earthly
+        cat <<EOF > ~/.earthly/config.yml
+        global:
+          buildkit_additional_config: |
+            [registry."docker.io"]
+              mirrors = ["193.148.176.216:5000"]
```

```suggestion
              mirrors = ["<FQDN>:5000"]
```
Can we use DNS here?

---

we gonna use a kubernetes service name here that will be deployed next to the runner, so the cache is only available internally